src/common/constants.ts

```diff
@@ -10,6 +10,10 @@ export const DEBUG_SERVER_SCRIPT_PATH = path.join(
   "tool",
   `_debug_server.py`,
 );
-export const RUFF_BIN_PATH = path.join(BUNDLED_PYTHON_SCRIPTS_DIR, "libs", "bin", "ruff");
+export const NEW_SERVER_SCRIPT_PATH = path.join(
```

Comment:
I'd vote `EXPERIMENTAL_SERVER_SCRIPT_PATH`, if that's what you mean by `NEW_`, so it's consistent with the setting? Easier to find later.